cli/root.go

```diff
@@ -78,6 +81,13 @@ func NewAlertmanagerClient(amURL *url.URL) *client.Alertmanager {
 
 	cr := clientruntime.New(address, path.Join(amURL.Path, defaultAmApiv2path), schemes)
 
+	if amURL.Scheme == "https" && skipVerify {
```

Comment:
Now I notice this. We should not check the scheme, but apply this all the time. You could go to an HTTP endpoint that redirects you (HTTP 302) to HTTPS.

Reply:
Didn't think we automatically followed them.  
Thanks, removed it.